src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -156,7 +156,7 @@ public static Command receive(Conveyor conveyor) {
   }
 
   /** Stop the conveor rollers. */
-  private static Command stop(Conveyor conveyor) {
+  public static Command stop(Conveyor conveyor) {
```

Comment:
Wait why should this be public now?

Reply:
before i added interrupt handling, Conveyor.stop() was only needed within the Conveyor class because the Conveyor's individual commands to receive and whatever called stop() on their own. Now that I need to stop the conveyor in StopMostThings, I need to make stop() public